packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -806,10 +803,9 @@ PageHeader.propTypes = {
    */
   subtitle: PropTypes.string,
   /**
-   * One or more tags, specified as a Carbon Tags component.
-   * Optional.
+   * An array of Carbon objects containing Carbon Tag properties plus 'label'. NOTE: 'filter' is not ignored.
    */
-  tags: PropTypes.arrayOf(PropTypes.element),
+  tags: TagSet.propTypes.tags,
```

Comment:
This is nice, BUT sadly doesn't show up well in Storybook etc. What we did with `actions` on Tearsheet and SidePanel was duplicate the proptypes definition from ActionSet -- messier, but it means the type shows up in the controls panel nicely.